io/sdp/src/main/java/org/mobicents/media/server/io/sdp/attributes/RtpMapAttribute.java

```diff
@@ -66,6 +66,7 @@
  */
 public class RtpMapAttribute extends AttributeField {
 	
+	private static final String NEWLINE = "\n";
```

Comment:
No need for this constant. Use System.lineSeparator() instead.
http://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--